docs/resources/aws_ec2_client_vpn_route.md

```diff
@@ -5,82 +5,82 @@ platform: aws
 
 # aws_ec2_client_vpn_route
 
-Use the `aws_ec2_client_vpn_route` InSpec audit resource to test properties of a single specific AWS EC2 Client VPN route.
+Use the `aws_ec2_client_vpn_route` InSpec audit resource to test the properties of a single specific AWS EC2 client VPN route.
 
-The `AWS::EC2::ClientVpnRoute` specifies a network route to add to a Client VPN endpoint.
+The `AWS::EC2::ClientVpnRoute` specifies a network route to add to a client VPN endpoint.
 
 ## Syntax
 
 Ensure that the client VPN endpoint exists.
 
-    describe aws_ec2_client_vpn_route(client_vpn_endpoint_id: "CLIENT_VPN_ENDPOINT_ID", target_subnet: "TARGET_SUBNET") do
+    describe aws_ec2_client_vpn_route(client_vpn_endpoint_id: "VPN_ID", target_subnet: "SUBNET_ID") do
       it { should exist }
     end
 
 ## Parameters
 
 `client_vpn_endpoint_id` _(required)_
 
-The ID of the Client VPN endpoint with which the route is associated.
+The association between the client VPN endpoint ID and the route.
```

Comment:
We should stick with the original text on this one as it comes directly from the AWS docs. 

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-clientvpnroute.html#cfn-ec2-clientvpnroute-clientvpnendpointid